src/utils/regex.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  address: /^[1-9]\d{0,19}L$/,
+  address: /^lsk[a-z0-9]{0,38}$/,
```

Comment:
Is this temporary? Cause we should test the address using Lisk Elements. A simple regex check won't suffice.

Reply:
ok then we'll have a dedicated PR for it.